src/spikeinterface/core/generate.py

```diff
@@ -317,6 +318,91 @@ def generate_snippets(
 
 
 ## spiketrain zone ##
+def synthesize_poisson_spike_vector(
+    num_units=20,
+    sampling_frequency=30000.0,
+    duration=60,
```

Comment:
```suggestion
    duration=60.0,
```
Since the docstring says it should be a float.